pkgs/tools/admin/exoscale-cli/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildGo114Package, fetchFromGitHub }:
+
+buildGo114Package {
+  pname = "exoscale-cli";
+  version = "1.12.0";
+
+  src = fetchFromGitHub {
+    owner  = "exoscale";
+    repo   = "cli";
+    rev    = "4b82b898b56ea238eb80ac38da40dc8a460a7de7";
+    sha256 = "04ym7mfv565icj3lmd2nrvq9asawwmmzg09681pj9py61ws56bxr";
+  };
+
+  goPackagePath = "github.com/exoscale/cli";
+  goDeps = ./deps.nix;
+
+  # ensures only the cli binary is built and we don't clutter bin/ with submodules
+  subPackages = [ "." ];
+
+  # we need to rename the resulting binary but can't use buildFlags with -o here
+  # because these are passed to "go install" which does not recognize -o
+  postBuild = ''
+    mv go/bin/cli go/bin/exo
```

Comment:
I see now what this is workarounding. I'm not sure about this. Could you please tell me what exactly did you try and what error / problem you hit? Also, just because I don't know, what does `-o` ?

Reply:
Ohh never mind, I understand it all now. It may sound like a stupid request, but would it work if you'd use:

```
postInstall = ''
mv $out/bin/cli $out/bin/exo
'';
```

It's just much more intuitive and readable IMO.

Reply:
`$out` doesn't exist `postInstall` - maybe something with the way go packages are built?

`ls: cannot access '/nix/store/g4ynb45r7w04phmwlw10mgqj0rhz59k8-exoscale-cli-1.12.0': No such file or directory`

Reply:
Yea, the `buildGoPackage` and `buildGoModule` functions are buggy and they are going through some refactoring these days. The comment is pretty good right now so :+1:.